exit

```diff
@@ -0,0 +1,377 @@
+[33mcommit c195c5a510b00978c8cd0fae65dcdf6b7abc1dbe[m[33m ([m[1;36mHEAD -> [m[1;32mmaster[m[33m, [m[1;31morigin/master[m[33m)[m
```

Comment:
Hey @md-abid-hussain 

It seems that this `exit` file is added mistakenly by you. You need to remove this file.